lib/workers/repository/init/merge.ts

```diff
@@ -212,7 +212,7 @@ export async function mergeRenovateConfig(
   }
   // Decrypt after resolving in case the preset contains npm authentication instead
   let resolvedConfig = await decryptConfig(
-    await presets.resolveConfigPresets(decryptedConfig, config),
+    await presets.resolveConfigPresets(decryptedConfig, config, [], [], true),
     repository
   );
```

Comment:
```suggestion
    await presets.resolveConfigPresets(decryptedConfig, config),
    repository
  );
```
see above.

Reply:
```suggestion
    await presets.resolveConfigPresets(decryptedConfig, true),
    repository
  );
```
as an alternative.
i think that if you add the flag as the third argument you can call it like so.

Reply:
as previously mentioned by @nabeelsaabna, we might want to explicitly set the flag value in the function invocation.
at least in non test files.
so the alternative might be better in this case.